src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -355,12 +355,18 @@ Copyright (c) .NET Foundation. All rights reserved.
                      Exists('$(AppHostSourcePath)')">
     <PropertyGroup>
       <_UseWindowsGraphicalUserInterface Condition="($(RuntimeIdentifier.StartsWith('win')) or $(DefaultAppHostRuntimeIdentifier.StartsWith('win'))) and '$(OutputType)'=='WinExe'">true</_UseWindowsGraphicalUserInterface>
+      <CreateAppHostRetries Condition="'$(CreateAppHostRetries)' == ''">3</CreateAppHostRetries>
+      <CreateAppHostRetryDelay Condition="'$(CreateAppHostRetryDelay)' == ''">1000</CreateAppHostRetryDelay>
```

Comment:
My instinct is to use `$(CopyRetryCount)` and `$(CopyRetryDelayMilliseconds)`--this feels very analogous. Or maybe as the defaults for the distinct properties?

Also, nit: usually this type of default property is defined outside a target.

Reply:
I think either reusing completely or reusing defaults would be fine.

Reply:
I'm fine with reusing the same properties.  I'll fix.

Reply:
Where do `CopyRetryCount` and `CopyRetryDelayMilliseconds` get defaulted?

I think we should be defaulting the apphost creation retry count to non-zero.

Reply:
I moved the defaulting of the properties out of the target and will reuse `CopyRetryCount` and `CopyRetryDelayMilliseconds` if I can find where they would be defaulted (and the defaults make sense for this use case).

Reply:
Looks like the values are assumed to be empty by default, and the copy task code is what defaults them. Good catch.

We could still reuse the same variables, but we would have to copy the defaults into the task. Maybe that's ok?

https://github.com/microsoft/msbuild/blob/acc2569a30fc9749942f4cdc58811e12acabf096/src/Tasks/Copy.cs#L76
https://github.com/microsoft/msbuild/blob/acc2569a30fc9749942f4cdc58811e12acabf096/src/Tasks/Copy.cs#L89

Reply:
That approach works for me.  I'll do that.